scripts/bb-dependencies.sh

```diff
@@ -13,19 +13,9 @@ fi
 # https://askubuntu.com/questions/132059/how-to-make-a-package-manager-wait-if-another-instance-of-apt-is-running
```

Comment:
I'd drop the reference here since you've simplied this down to a trival wait loop.